Windows/MainWindow.xaml.cs

```diff
@@ -97,7 +97,7 @@ public bool ShowIdBeforeName {
             }
         }
 
-        public bool SingleClickToEditMode { get; set; } = true;
+        public bool SingleClickToEditMode { get { return UpdateCheck.SingleClick_Mode; } set { UpdateCheck.SingleClick_Mode = value; } }
```

Comment:
Put the public static here; the line above.
See `ShowIdBeforeName` just above. The backing field is just right above it. There will only ever be one instance of this window, so doing it this way is fine, and consistent with the rest static settings fields.